deploy/eks-terraform/README.md

```diff
@@ -0,0 +1,17 @@
+# Sample deployment of kiam on AWS EKS
+This is a sample deployment of kiam on Amazon EKS with minimal dependencies.
+
+## Dependencies
+
+- terraform for creating the resources on AWS
+- aws cli
+- aws-iam-authenticator for authenticating to the cluster
+- kubectl
+- cfssl
+- basic linux tools
```

Comment:
could you add links to these tools to make it easier to install?
e.g [cfssl](https://github.com/cloudflare/cfssl).

Reply:
Sure. aws-iam-authenticator is not even required anymore since the aws cli can do that now.
I should be able to do it this week.